spec/features/reservations_spec.rb

```diff
@@ -552,7 +552,6 @@
       select('10', from: 'items_per_page') # show more than 1 item (default)
```

Comment:
Should we just make this the default for our own sanity moving forward? It's all nice and well that we finally figured out what was causing this bug, but I feel like defaulting to 10/page (probably in the app config factory) would just make our specs more robust. What do you think?